pyproject.toml

```diff
@@ -26,7 +25,6 @@ classifiers = [
   "Typing :: Typed",
 ]
 requires-python = ">= 3.11, < 4"
-# TODO(cookiecutter): Remove and add more dependencies if appropriate
```

Comment:
Oh, OK, the dependency to `frequenz-api-dispatch` is not added yet. Ignore the comment about the cross-reference if you don't plan to add it in this PR.

Reply:
I suppose I should do it in this PR else i might forget all these places to reference it

Reply:
Yeah, I agree it makes sense to include the actual work to remove the TODOs in this initial PR. :+1: